db/migrate/20181211094456_add_indexes_for_send_emails.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexesForSendEmails < ActiveRecord::Migration[5.1]
+  def change
+    add_index :listings, [:community_id, :author_id, :deleted], name: 'person_community_exist'
```

Comment:
This index is not strictly needed, I think, but let's keep it. As far as I could check, queries that join listings, use community_id in the join, which was enough so that the index on (community_id, person_id) is used. Name could be perhaps `community_author_deleted`. This branch was already deployed to staging, right? What's the best way to change a migration then?